src/onlyspans.polyleads.ui/src/components/chat-bot/request-examples-cards/request-examples-cards.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+} from '@/components/ui/carousel';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { requestExamplesData } from '@/components/chat-bot/request-examples-cards/request-examples.data';
+import Autoplay from 'embla-carousel-autoplay';
+import useGet from '@/hooks/useGet';
+import { IRequestFormVM } from '@/components/chat-bot/request-form/request-form.vm';
+import ServiceSymbols from '@/data/constant/ServiceSymbols';
+
+const RequestExamplesCards: FC = () => {
+  const vm = useGet<IRequestFormVM>(ServiceSymbols.IRequestFormVM);
+
+  return (
+    <Carousel
+      opts={{
+        align: 'start',
+      }}
+      plugins={[
+        Autoplay({
+          delay: 2000,
+        }),
+      ]}
+      className='lg:w-[75%] sm:w-[70%] w-[50%]'
+    >
+      <CarouselContent>
+        {requestExamplesData.map((example, index) => (
```

Comment:
```suggestion
        {promptSamplesData.map((sample, index) => (
```

в других местах тоже поменяй пж